wfe/probs.go

```diff
@@ -25,6 +25,8 @@ func problemDetailsForBoulderError(err *berrors.BoulderError, msg string) *probs
 		return probs.RejectedIdentifier(fmt.Sprintf("%s :: %s", msg, err))
 	case berrors.InvalidEmail:
 		return probs.InvalidEmail(fmt.Sprintf("%s :: %s", msg, err))
+	case berrors.CAA:
```

Comment:
This diff also needs to be applied to the `wfe2.problemDetailsForBoulderError` function in `wfe2/probs.go`

Reply:
Friendly bump on this requested change. 72923d8 only fixed CI.